compiler/aarch64/codegen/ARM64Debug.cpp

```diff
@@ -860,6 +880,11 @@ TR_Debug::print(TR::FILE *pOutFile, TR::ARM64Trg1ImmInstruction *instr)
          }
       trfprintf(pOutFile, ", " POINTER_PRINTF_FORMAT, (instr->getBinaryEncoding() + offset));
       }
+   else if ((op == TR::InstOpCode::fmovimms) || (op == TR::InstOpCode::fmovimmd))
+      {
+      uint32_t imm = instr->getSourceImmediate() & 0xFF;
+      trfprintf(pOutFile, ", %lf", getDoubleFromImm8(imm));
```

Comment:
Is it useful to print the immediate in addition to the conversion to double?  Yes, the conversion is convenient, but sometimes you may want to know exactly what value was encoded in the instruction.

Reply:
I updated the code and it now prints like this:
```
fmovimmd s1, 0x60 (0.500000)
```